sys/vm/vm_map.c

```diff
@@ -2771,6 +2771,18 @@ vm_map_madvise(
 			if (current->eflags & MAP_ENTRY_IS_SUB_MAP)
 				continue;
 
+			/*
+			 * MADV_FREE would otherwise rewind time to the
+			 * creation of the shadow object.  This is a
+			 * temporary fix to
+			 * https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=240061
+			 */
+			if ((behav == MADV_FREE)
```

Comment:
Style is `&&` at the end of the line and no parens around comparison expressions.

Reply:
Thanks